test/gtest/solver_bwd.hpp

```diff
@@ -66,12 +80,11 @@ struct ConvBwdSolverTest
             return tmp;
         }();
 
-        // const auto network_config = problem.BuildConfKey();
-
         if(!solv.IsApplicable(ctx, problem))
         {
-            test_skipped = true;
-            GTEST_SKIP() << solv.SolverDbId() << ": Not Applicable for this problem" << conv_config;
+            // Do not put GTEST_SKIP here! An inappropriate config should cause the test to fail,
+            // not skip. Otherwise, such testing is pointless.
```

Comment:
We should keep `GTEST_SKIP` mentioned here, because it is a fundamental point. We have a bunch of tests that are simply skipped with inapplicable configs.